includes/wikia/services/HubService.class.php

```diff
@@ -61,6 +81,31 @@ public static function getCurrentWikiaVerticalName() {
 	}
 
 	/**
+	 * Get canonical vertical name for given cityId.
+	 * For Lifestyle and Gaming return their names.
+	 * For rest of values return Entertainment.
+	 * @param $cityId
+	 * @return Boolean|String
+	 */
+	public static function getVerticalNameForComscore($cityId) {
+		$verticalId = WikiFactoryHub::getInstance()->getVerticalId($cityId);
+		$canonicalVerticalId = self::getCanonicalVerticalID($verticalId);
+
+		switch ($canonicalVerticalId) {
```

Comment:
I added the logic to fallback to LIFESTYLE if vertical is not set.
What do you mean by double checking the names?
I would like to simplify the logic as much as possible.
From what I understood from Amber's comment, we don't want to fallback to old category.
It is either GAMING (for gaming hub), ENTERTAINMENT (for everything beside gaming and lifestyle) or LIFESTYLE (for lifestyle or unset vertical).